### DISCUSSION TITLE
Skill Provisioning failed. Please create another Alexa Hosted Skill.

### DISCUSSION THREAD

**Discussion by Mike144007** [keatontaylor/alexa-actions#232] [General]
Hi there
Can anyone tell me whether anything has changed regarding the way the Alexa Developer Console builds skills / models ?
I have been trying to follow the wiki for alexa actionable notifications in home assistant, however I am getting different results to those suggested. Initially, when testing if the skill is addressable, I get a 401 error that Home Assistant is not reachable (not surprising as at this point no long lived access token has been entered, but still different from the suggested response in the wiki). Then when I try to configure the python code, I am getting the message in the title - skill provisioning failed. 
I have in the past had attempts where this message did not come up, although I still failed to complete the whole process for some reason or another. 
So is it just me ? As far as I can tell I am following the instructions correctly.

**Comment by DEADSEC-SECURITY** [comment]
Could you share some screenshots of the message? That seems like an issue with your configuration of the skill or AWS itself.

**Comment by DEADSEC-SECURITY** [comment]
Seems like there is a solution https://github.com/keatontaylor/alexa-actions/issues/233#issuecomment-1947431532.

Check that and let me know

**Reply by DEADSEC-SECURITY** [reply]
Another issue related: https://github.com/keatontaylor/alexa-actions/issues/234

**Reply by Krispkiwi** [reply]
Was there any luck on this today? The 233 comment doesn't solve it.

**Comment by Mike144007** [comment]
Hi guys
Really appreciate you trying to help. Sorry I have not had a chance to take this any further for the last day or so. Will respond properly when I get some time.

**Comment by DEADSEC-SECURITY** [comment]
Moving this to https://github.com/keatontaylor/alexa-actions/issues/233 as its a know bug now
